creator-node/src/services/sync/secondarySyncFromPrimary.ts

```diff
@@ -598,26 +598,29 @@ const handleSyncFromPrimary = async ({
                 nonTrackFile.type === 'image' &&
                 nonTrackFile.fileName !== null
               ) {
-                error = await saveFileForMultihashToFS(
+                const { error: fetchError } = await fetchFileFromNetworkAndSaveToFS(
                   libs,
                   logger,
                   multihash,
-                  nonTrackFile.storagePath,
+                  nonTrackFile.dirMultihash,
                   gatewaysToTry,
                   nonTrackFile.fileName
                 )
+                error = fetchError
               } else {
-                error = await saveFileForMultihashToFS(
+                const { error: fetchError } = await fetchFileFromNetworkAndSaveToFS(
                   libs,
                   logger,
                   multihash,
-                  nonTrackFile.storagePath,
+                  nonTrackFile.dirMultihash,
                   gatewaysToTry
                 )
+                error = fetchError
               }
 
               // If saveFile op failed, record CID for later processing
               if (error) {
+                console.error('secondarySyncFromPrimary error', error)
```

Comment:
probably should delete or make this an info log

Reply:
fixed